repo2docker/buildpacks/conda/__init__.py

```diff
@@ -81,7 +109,7 @@ def get_build_scripts(self):
                 r"""
                 TIMEFORMAT='time: %3R' \
                 bash -c 'time /tmp/install-miniforge.bash' && \
-                rm /tmp/install-miniforge.bash /tmp/environment.yml
+                rm -rf /tmp/install-miniforge.bash /tmp/env
```

Comment:
Question: is it worth removing the environment spec? These are quite small files and might be nice to have as remnants in the images for debugging.

Reply:
Good call. Let's clean them up for now since they are in tmp, and we can open a separate PR to persist them to a non-tmp location if we decide that makes senes.